sdks/java/testing/tpcds/build.gradle

```diff
@@ -56,9 +56,6 @@ configurations {
 dependencies {
     implementation enforcedPlatform(library.java.google_cloud_platform_libraries_bom)
 
-    // TODO(BEAM-13179): Determine how to build without this dependency
-    compileOnly "org.immutables:value:2.8.8"
```

Comment:
There are a few places where this exists beyond this module. I tried removing each of them as well during the Gradle 7 migration but was unsuccessful in getting the code to compile without it.